core/services/job/job_orm_test.go

```diff
@@ -849,6 +851,57 @@ func Test_FindPipelineRunIDsByJobID(t *testing.T) {
 
 		assert.Equal(t, run.ID, runIDs[0])
 	})
+
+	jobs := make([]job.Job, 10)
+	for j := 0; j < len(jobs); j++ {
+		_, bridge := cltest.MustCreateBridge(t, db, cltest.BridgeOpts{}, config)
+		_, bridge2 := cltest.MustCreateBridge(t, db, cltest.BridgeOpts{}, config)
```

Comment:
Is it necessary to create separate bridges for each job?

Reply:
If I recall, I used the same bridge for all in my first attempt, but trying to create the job failed with an error about a duplicate bridge.  I'll double check on that, might be the way in which these cltest functions are implemented.